ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_internet_exit.py

```diff
@@ -35,13 +35,10 @@ def router_internet_exit(self) -> dict | None:
             # TODO: Today we use the order of the connection list to order the exit-groups inside the policy.
             #       This works for zscaler but later we may need to use some sorting intelligence as order matters.
             for connection in policy.get("connections", []):
-                if connection["type"] == "tunnel":
-                    exit_group_name = connection["exit_group"]
-                    exit_groups_dict.setdefault(exit_group_name, {"local_connections": []})["local_connections"].append(
-                        {"name": f"IE-Tunnel{connection['tunnel_id']}"}
-                    )
-                    # Recording the exit_group in the policy
-                    policy_exit_groups.append(exit_group_name)
+                exit_group_name = connection["exit_group"]
+                exit_groups_dict.setdefault(exit_group_name, {"local_connections": []})["local_connections"].append({"name": connection["name"]})
+                # Recording the exit_group in the policy
+                policy_exit_groups.append(exit_group_name)
```

Comment:
so we probably need to change this for an ordered set?

Reply:
or just check before appending.